subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/PluginConfigurationAttributesIntegrationTest.groovy

```diff
@@ -59,9 +87,6 @@ class PluginConfigurationAttributesIntegrationTest extends AbstractIntegrationSp
         'jacoco'     | 'jacocoAnt'
         'pmd'        | 'pmd'
         'checkstyle' | 'checkstyle'
-        'scala'      | 'zinc'
```

Comment:
How come these were removed?

Reply:
I think there were some merging difficulties here.  There's no need for 2 test methods, I'll combine them.  Good catch!